### DISCUSSION TITLE
Clickable Link

### DISCUSSION THREAD

**Discussion by hibaq86** [FlowiseAI/Flowise#1574] [Q&A]
How do I do hyperlinks in the data so that people can click on it? Also create references for each answer to a clickable link?

**Comment by jeffh86** [comment]
I have created links in my chatbots. I used cherrio web scraper and added to the prompt in conversational retrieval QA chain "You will need to use urls in your answers to help achieve your goal. These links must be clickable."

I also added a list of urls from the xml sitemap of the site, as it came back with some strange ones before doing this.

The only thing I would add is that sometimes they come back in a great format. However, sometimes the whole url is in the response and it does not fit in the box and looks not great.

For example, a web address might be https://exmaple.com/seo and sometimes it gives the whole url, and other times it has a click "here link" or just "SEO". I have tried all sorts of prompts and can't get it to do this second method all the time, which is annoying.

**Comment by Jaredude** [comment]
The Flowise chatbot supports [markdown](https://www.markdownguide.org/cheat-sheet/) You'd structure the link the same as on GitHub
`[markdown](https://www.markdownguide.org/cheat-sheet/)`
